graylog2-server/src/main/java/org/graylog2/plugin/indexer/searches/timeranges/AbsoluteRange.java

```diff
@@ -115,14 +114,13 @@ private DateTime parseDateTime(String s) {
                 throw new IllegalArgumentException("Null or empty string");
             }
 
-            final DateTimeFormatter formatter;
```

Comment:
I think this change to `AbsoluteRange.java` should not be in there. We need to do some more careful research & testing to ensure we can do this.

Reply:
You're right, I forgot to remove that. It's reverted now.